katello/katello-debug.sh

```diff
@@ -126,9 +126,19 @@ add_cmd "rpm -qa | grep qpid" "qpid-rpm-qa"
 add_cmd "mongo pulp_database --eval \"db.reserved_resources.find().pretty().shellPrint()\"" "mongo-reserved_resources"
 add_cmd "mongo pulp_database --eval \"db.task_status.find().pretty().shellPrint()\"" "mongo-task_status"
 
-echo "db.task_status.find({state:{\$ne: \"finished\"}}).pretty().shellPrint()" > /tmp/pulp_running_tasks.js
+TEMP_DIR=`mktemp -d`
+
+echo "db.task_status.find({state:{\$ne: \"finished\"}}).pretty().shellPrint()" > $TEMP_DIR/pulp_running_tasks.js
+
+add_cmd "mongo pulp_database $TEMP_DIR/pulp_running_tasks.js" "pulp-running_tasks"
+
+cleanup() {
+  rm $TEMP_DIR/pulp_running_tasks.js
+  rmdir $TEMP_DIR
+}
+trap "cleanup" SIGHUP SIGINT SIGTERM EXIT
```

Comment:
This line needs to be above the `echo` otherwise you can still experience uncleaned temp file.